src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
```

Comment:
add `font-style` and `font-weight` styles

Reply:
completed